src/views/Receiving.vue

```diff
@@ -5,85 +5,80 @@
-              <p>2 {{ $t("items") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-          <ion-item>
-            <ion-label>
-              <h2>SHPMNT_003</h2>
-              <p>6 {{ $t("items") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-          <ion-item>
-            <ion-label>
-              <h2>SHPMNT_004</h2>
-              <p>4 {{ $t("items") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-          <ion-item>
-            <ion-label>
-              <h2>SHPMNT_005</h2>
-              <p>1 {{ $t("item") }}</p>
-            </ion-label>
-            <ion-note slot="end">{{ $t("Shipped") }}</ion-note>
-          </ion-item>
-        </ion-list>
+        <ion-searchbar :placeholder="$t('Scan ASN to start receiving')"/>
+
+        <shipment-list-item v-for="shipment in shipments" :key="shipment.productId" :shipment="shipment"/>
+
         <div class="ion-text-center">
-          <ion-button fill="outline" color="dark"><ion-icon :icon="cloudDownloadOutline" slot="start" />{{ $t("Load more shipments") }}</ion-button>
+          <ion-button fill="outline" color="dark"><ion-icon :icon="cloudDownloadOutline" slot="start" @click="loadMoreShipments()" />{{ $t("Load more shipments") }}</ion-button>
+        </div>
         </div>
-      </div> 
     </ion-content>
   </ion-page>
 </template>
 
 <script lang="ts">
-import { IonButton, IonContent, IonHeader, IonIcon, IonItem, IonLabel, IonList, IonNote, IonPage, IonSearchbar, IonTitle, IonToolbar } from '@ionic/vue';
-import { defineComponent } from 'vue';
+import { IonButton, IonContent, IonHeader, IonIcon, IonPage, IonSearchbar, IonTitle, IonToolbar } from '@ionic/vue';
 import { cloudDownloadOutline } from 'ionicons/icons'
-import { useRouter } from 'vue-router';
+import { defineComponent } from 'vue'
+import { mapGetters, useStore } from 'vuex'
+import ShipmentListItem from '@/components/ShipmentListItem.vue'
 
 export default defineComponent({
-  name: 'Receiving',
+  name: "Receiving",
   components: {
-    IonButton,
+     IonButton,
     IonContent,
     IonHeader,
     IonIcon,
-    IonItem,
-    IonLabel,
-    IonList,
-    IonNote,
     IonSearchbar,
     IonPage,
     IonTitle,
-    IonToolbar
+    IonToolbar,
+    ShipmentListItem
   },
-  setup(){
-    const router = useRouter();
+  computed: {
+    ...mapGetters({
+      shipments: 'shipment/getSearchProducts',
+      user: 'user/getCurrentFacility'
+    })
+  },
+  data(){
+    return{
+      viewSize:'',
+      viewIndex:''
+    }
+  },
+  mounted () {
+    this.getShipments(null, null);
+  },
+  methods: {
+    selectSearchBarText(event: any) {
+      event.target.getInputElement().then((element: any) => {
+        element.select();
+      })
+    },
+    async getShipments(vSize: any, vIndex: any) {
```

Comment:
We can make `vSize` and, `vIndex` as optional parameters.

Reply:
Done